dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/test/com/ibm/ws/jaxrs/utils/test/JAXRSOutInterceptorTest.java

```diff
@@ -18,16 +18,14 @@
 import java.util.Date;
 import java.util.Locale;
 
-import org.apache.cxf.jaxrs.interceptor.CachedTimeAccessor;
 import org.apache.cxf.jaxrs.utils.HttpUtils;
 import org.junit.Test;
 
-import junit.framework.Assert;
-
+/**
```

Comment:
Same here.  My last PR is completely undone.  I can make the PR to put my change back if you want.